pgx-utils/src/sql_entity_graph/to_sql.rs

```diff
@@ -0,0 +1,149 @@
+use std::hash::Hash;
+
+use proc_macro2::TokenStream as TokenStream2;
+use quote::{quote, ToTokens, TokenStreamExt};
+use syn::spanned::Spanned;
+use syn::{AttrStyle, Attribute, Lit, Meta, MetaList, MetaNameValue, NestedMeta};
+
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub struct ToSqlConfig {
+    pub enabled: bool,
+    pub callback: Option<syn::Path>,
+    pub content: Option<syn::LitStr>,
+}
+impl From<bool> for ToSqlConfig {
+    fn from(enabled: bool) -> Self {
+        Self {
+            enabled,
+            callback: None,
+            content: None,
+        }
+    }
+}
+impl From<syn::LitStr> for ToSqlConfig {
+    fn from(content: syn::LitStr) -> Self {
+        if let Ok(path) = content.parse::<syn::Path>() {
+            return Self {
+                enabled: true,
+                callback: Some(path),
+                content: None,
+            };
+        } else {
+            return Self {
+                enabled: true,
+                callback: None,
+                content: Some(content),
+            };
+        }
+    }
+}
+impl Default for ToSqlConfig {
+    fn default() -> Self {
+        Self {
+            enabled: true,
+            callback: None,
+            content: None,
+        }
+    }
+}
+
+const INVALID_ATTR_CONTENT: &str =
+    "expected `#[pgx(sql = content)]`, where `content` is a boolean, string, or path to a function";
```

Comment:
You mention a function path here but I don't see code to handle it. :(

Reply:
It's on line 79, we expect the path to be provided as a string